lib/eslint.js

```diff
@@ -757,6 +757,8 @@ module.exports = (function() {
      */
     api.verify = function(textOrSourceCode, config, filenameOrOptions, saveState) {
         const text = (typeof textOrSourceCode === "string") ? textOrSourceCode : null;
+        const selectors = new Set();
```

Comment:
The one I was talking about is `EventEmitter._events` property. It's available in every version (to the best of my knowledge), but it's not documented and private. So not sure if that's a great idea to use it.